src/pages/docs/octopus-cloud/index.mdx

```diff
@@ -14,104 +14,55 @@ import OctopusCloudInstance from 'src/shared-content/octopus-cloud/octopus-cloud
+## Octopus Cloud storage thresholds \{#octopus-cloud-storage-limits}
+Octopus Cloud instances are subject to storage thresholds.
 
-<OctopusCloudInstance />
+- Maximum file storage for artifacts, task logs, packages, package cache, and event exports is limited to 1 TB.
+- Maximum database size for configuration data (for example, projects, deployment processes, and inline scripts) is limited to 100 GB.
+- Maximum size for any single package is 5 GB.
+- [Retention policies](/docs/administration/retention-policies) default to 30 days, but you can change this figure as needed.
 
-When the instance is ready, you will see it (and any other instances you have access to) the next time you log in to your Octopus account at [https://octopus.com/signin](https://octopus.com/signin).
-
-## Where is Octopus Cloud hosted \{#octopus-cloud-hosting-locations}
-
-<OctopusCloudRegions />
-
-## Octopus Cloud storage limits \{#octopus-cloud-storage-limits}
-
-Octopus Cloud instances are subject to storage limits. Specifically:
-
-<OctopusCloudStorageLimits />
+Enterprise customers can exceed these limits. Learn more by [contacting Sales](https://octopus.com/company/contact).
 
 Please see our [Cloud pricing FAQ](https://octopus.com/pricing/faq#are-there-any-storage-limits) for further details.
 
-## Uploading packages to Octopus Cloud servers \{#uploading-packages-to-octopus-cloud-servers}
-
-Factors such as geographical distance, network bandwidth, and network congestion may cause package uploads to your Octopus Cloud Server to take longer than expected.
-
-If you are having difficulty uploading packages to your Octopus built-in package feed within the default timeout threshold, typically within 5 minutes, you might be affected by one or more of the factors mentioned above. In this case, you could try increasing the timeout threshold of the Octopus CLI or `nuget.exe` to a value that suits you. We also recommend using the Octopus CLI, **Octopus.Client**, or the **TeamCity Plugin (v4.41.0+)** because they have the advantage of using delta compression.
-
-## Change your password \{#change-your-password}
-
-To change your password for the Octopus instance and Octopus account:
-
-1. Go to [Octopus.com/signin](https://Octopus.com/signin).
-1. Click the drop-down menu next to your username in the top right corner.
-1. Click profile.
-1. Click the **Change Password** link.
-1. Enter your new password.
-1. Confirm the new password, and click **Change password**.
-
-## Reset the Octopus account password \{#reset-the-octopus-account-password}
-
-If you forget your username or password for your Octopus account or Octopus instance, you can reset them via Octopus.com
-
-1. Visit [Octopus.com/signin](https://Octopus.com/signin).
-1. Click **Forgot your password?**
-1. Reset your credentials and log into your Octopus account. You can now launch your Octopus instance from the accounts dashboard.
-
-## Invite users to your Octopus Cloud instance \{#inviting-octopus-cloud-users}
```

Comment:
This bookmark is referenced in short.io / the Octopus product, so the bookmark test is failing.